internal/healthchecks/api_check.go

```diff
@@ -94,10 +94,11 @@ func monitoringPing(ctx context.Context, client monitoring.MetricClient, gceMeta
 func runLoggingCheck(logger *log.Logger) error {
 	ctx := context.Background()
 	gceMetadata, err := getGCEMetadata()
+
 	if err != nil {
 		return fmt.Errorf("can't get GCE metadata: %w", err)
 	}
-	logger.Printf("gce metadata: %+v", gceMetadata)
+	logger.Printf("GCE Metadata queried successfully")
```

Comment:
[Optional] Do we want to print some sentinel value, rather than just indicating success?